pubspec.lock

```diff
@@ -59,21 +59,28 @@ packages:
       name: mapbox_gl_dart
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.0-nullsafety.0"
+    version: "0.2.1"
   mapbox_gl_platform_interface:
     dependency: "direct main"
     description:
       path: mapbox_gl_platform_interface
       relative: true
     source: path
-    version: "0.14.0"
+    version: "0.15.0"
   mapbox_gl_web:
     dependency: "direct main"
     description:
       path: mapbox_gl_web
       relative: true
     source: path
-    version: "0.14.0"
+    version: "0.15.0"
+  material_color_utilities:
```

Comment:
where is this coming from?

Reply:
I think this is auto-generated when I run the project